docs/src/extend/custom-rules.md

```diff
@@ -787,6 +789,47 @@ module.exports = {
 
 To learn more about JSON Schema, we recommend looking at some examples on the [JSON Schema website](https://json-schema.org/learn/), or reading the free [Understanding JSON Schema](https://json-schema.org/understanding-json-schema/) ebook.
 
+#### Option Defaults
```

Comment:
Should we mention how this interacts with `default` values from the schema and which one is recommended?